pkg/drivers/kvm/domain.go

```diff
@@ -99,6 +102,37 @@ $ newgrp libvirt
 Visit https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#kvm-driver for more information.
 `
 
+func randomMAC() ([]byte, error) {
+	buf := make([]byte, 6)
+	_, err := rand.Read(buf)
+	if err != nil {
+		return nil, err
+	}
+	// We unset the first and second least significant bits (LSB) of the MAC
+	//
+	// The LSB of the first octet
+	// 0 for unicast
+	// 1 for multicast
+	//
+	// The second LSB of the first octet
+	// 0 for universally administered addresses
+	// 1 for locally administered addresses
+	buf[0] = buf[0] & 0xfc
+	return buf, nil
+}
+
+func macToString(buf []byte) string {
```

Comment:
You might be able to get this for free by using net/HardwareAddress from the stdlib: https://golang.org/src/net/mac.go

It looks like it takes a byte slice to construct it.

Reply:
Nice, that really simplified it

Reply:
done